internal/exec/stages/files/files.go

```diff
@@ -164,6 +152,11 @@ func (tmp dirEntry) create(l *log.Logger, u util.Util) error {
 			return err
 		}
 
+		uid, gid, err := u.GetNodeOwners(d.Node, 0, 0)
```

Comment:
Does this have the same functionality as previously? Before it was possible to have the function return an error if either the User.ID or Group.ID was nil which wouldn't hit the `u.DeletePathOnOverwrite` function.

Reply:
It does not. It relies on `d.Node.User` (or `.Group`) being nil, or having exactly one of its two fields set (validation should ensure this). If `id` is set, it returns that, if `name` is set it does the lookup to resolve the `uid` or `gid`, if `d.Node` is nil it returns the provided defaults.

The only way it can error is if an error is encountered resolving the `name` field for the user or group.